Docs: Enhance `stream_analytics_job` example to use `StandardV2` SKU and add details around `streaming_units` parameter (#27037)
hashicorp/terraform-provider-azurerm: website/docs/r/stream_analytics_job.html.markdown

* `output_error_policy` - (Optional) Specifies the policy which should be applied to events which arrive at the output and cannot be written to the external storage due to being malformed (such as missing column values, column values of wrong type or size). Possible values are `Drop` and `Stop`. Default is `Drop`.

> * `streaming_units` - (Required when type is set to its default of `Cloud` but otherwise optional) Specifies the number of streaming units that the streaming job uses. Supported values are `1`, `3`, `6` and multiples of `6` up to `120`. For a conversion table what the value would be for V2 units, please see docs here: https://learn.microsoft.com/azure/stream-analytics/stream-analytics-streaming-unit-consumption#understand-streaming-unit-conversions-and-where-they-apply

Reviewer: The value in the parentheses behind the property name should always reflect what has been set in the schema which is `Optional` for this property. Special behaviours are expanded on in the property description or in a `Note` below the property as has been done here:

https://github.com/hashicorp/terraform-provider-azurerm/blob/8ab553cac61c929acf4992558a06d69dff7c92f4/website/docs/r/stream_analytics_job.html.markdown?plain=1#L80

Could we update this to
```suggestion
* `streaming_units` - (Optional) Specifies the number of streaming units that the streaming job uses. Supported values are `1`, `3`, `6` and multiples of `6` up to `120`. A conversion table for V2 streaming units can be found [here](https://learn.microsoft.com/azure/stream-analytics/stream-analytics-streaming-unit-consumption#understand-streaming-unit-conversions-and-where-they-apply)
```
